VaticanBAV/et/BAVet119.xml

```diff
@@ -0,0 +1,552 @@
+                          </msItem>
+                          <msItem xml:id="ms_i1.3">
+                              <locus from="4r" to="11v"/>
+                              <title type="complete" ref="LIT1171Awdana#tables">Sixteen circles, each divided into sixteen parts.</title>
+                              <incipit xml:lang="gez">
+                                  ፩ ዓውደ፡ ሆይ። ሀ ፩ትረክቦ፡ ለዜና፡ መፍቅድ፡ በባሕረ፡ ፃና።
+                              </incipit>
+                          </msItem>
+                          <msItem xml:id="ms_i1.4">
+                              <locus from="13r" to="36v" />
+                              <title type="complete" ref="LIT1171Awdana#seas">Sixteen chapters, indicated with the names of local lakes:
+                                  Ṣānā, Kʷarā, ʾElā, Šaklā, Zʷāy, ʾAlzazo, Wənǧ, <sic resp="MV">Goǧǧam</sic>, Dāgo, 
+                                  Ḥayq, Heno, Šemazbe, Ḥawaš, Gəb, Takaze, ʾAbāwi.</title>
+                              <incipit xml:lang="gez">
+                                  በስመ፡ አብ፡ <gap reason="ellipsis"/> ፩ አምላክ፡ መሐሪ፡ ወመስተሣህል፡ ዘተናገረ፡ ፩ ጠቢ<surplus>ቢ</surplus>ብ፡ እምነ፡ ብዙኃን፡ ጠቢባን፡ ወፈላስፋውያን፡
+                                  ዘንተ፡ ዘስሙ፡ መጽሐፈ፡ ምክር፡ ለአበው፡  <sic>ቀደመት፡</sic> ከመ፡ ያእምሩ፡ በቱ፡ በዝ፡ እምሂ፡ ሠናየ፡ ወዕመሂ፡ እኩየ፡
+                                  <surplus>ለ</surplus>ሰብአ፡ ዓለም፡ ለለርእሶሙ፡ ውእቱ፡ ፩ በኃሢሠ፡ መፍቅድ፡ ሐቲት፡ ፪ ሐቲት፡ በነጊደ፡ ንዋይ፡ ፫ ሐቲት፡ 
+                                  <supplied reason="omitted">በ</supplied>በዊዓ፡ ቤተ፡ ንጉሥ፡ <supplied reason="omitted"/> ፩ <del rend="expunctuated">ባሕረ፡ ፃና</del>
+                                  ፩ በኃሢሠ፡ መፍቅድ፡ ናሁ<supplied reason="omitted">፡</supplied> መጽአ፡ ለከ፡ ሠናይ፡ ዘያስተፌሥሕ፡ ልበከ፡ ብስራት፡ ለከ፡ እስመ፡ ይትፌጸም፡ ጻሕቅከ፡
+                              </incipit>                         
+                          <explicit xml:lang="gez">
+                              ፲፮ ሐቲ<supplied reason="omitted">ት፡</supplied> በተዋስቦ፡ እመሰ፡ ትፈቅድ፡ ተዋስቦ፡ ተዓገሥ፡ እስመ፡ <sic>በተዓገሦ፡</sic> ይትረከብ፡
+                              በ<del rend="expunctuated">ብ</del>ትፍሥሕት፡ <sic>ወኃሢት፤</sic> ወብዕል፡ ወኵሉ፡ <sic>ተምኒትከ፡</sic> ወአንተ፡ <sic>ኵን፡</sic> ኅዱዓ፡
+                              በኵሉ፡ <choice><sic>ነገር፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ነገረ፡</corr></choice> ተዋስቦ፡ ወዝኒ፡ ተዋስቦ፡ ዘትሄሊ፡ ይከውኖሙ፡
+                              ለእሉ፡ ትፍሥሕት፡ <sic>ወኃሢት፡</sic> ወብዕል፨
+                              </explicit>
+                      </msItem>
+                      </msItem>
+                      
+                      <msItem xml:id="ms_i2">
+                          <locus from="37r" to="54v"/>
+                          <title type="complete" ref="LIT7107Divination">Treatise on divination</title>
+                          <textLang mainLang="gez"/>
+                          <msItem xml:id="ms_i2.1">
+                              <locus from="37r" to="37v"/>
+                              <title type="complete" ref="LIT7107Divination#table1">Table divided into sixteen columns and seven orders.</title>
```

Comment:
If `LIT7107Divination#table1` has `Table divided into sixteen columns and seven orders` as its `label `you do not need to provide it here.

Reply:
It should be: Table divided into sixteen columns and seven **lines**.